### DISCUSSION TITLE
Is there a way i can know the rotation angle between the equatorial and azimuth mount?

### DISCUSSION THREAD

**Discussion by JyKlub** [Stellarium/stellarium#3978] [Q&A]
Hi, i have been browsing through the Q&As to look for a way to rotate the view or the orientation so as to adjust to camera view, pretty much the same with issue 3705: [https://github.com/Stellarium/stellarium/issues/3705](https://github.com/Stellarium/stellarium/issues/3705), as far as i ve found out it still has to be done manually i suppose? 
Meanwhile switching between equatorial and azimuth mount is possible:
with equatorial mount:
![image](https://github.com/user-attachments/assets/8815550d-b269-490c-833b-ff8cc151818c)
with azimuth mount:
![image](https://github.com/user-attachments/assets/84b24645-b2d0-4b96-bd49-44d605bb3709)
I would like to know the rotational angle between these two frames(or how much the the red camera rectangle rotated). Of course i can calculate this value with some post processing but would be helpful if i cant access it directly. Also is this angle fixed? Thanks

**Comment by gzotti** [comment]
The parallactic angle is available at least in the infomap accessible from Scripting.

**Reply by JyKlub** [reply]
Thanks! Is the function _getParallacticAngle_ available in the script console? How can i call it? I tried core.getParallacticAngle but it's not recognized
